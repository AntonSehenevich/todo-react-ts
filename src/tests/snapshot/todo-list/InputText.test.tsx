import renderer from 'react-test-renderer'
import TextInput from '../../../components/todos/text-input'

test('renders correctly', () => {
  const tree = renderer.create(<TextInput onSave={jest.fn()} />).toJSON()

  expect(tree).toMatchSnapshot()
})
