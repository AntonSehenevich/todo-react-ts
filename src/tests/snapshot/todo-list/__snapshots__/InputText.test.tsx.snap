// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<input
  className="input"
  onBlur={[Function]}
  onChange={[Function]}
  onKeyDown={[Function]}
  placeholder=""
  type="text"
  value=""
/>
`;
